crates/fj-kernel/src/algorithms/transform/sketch.rs

```diff
@@ -7,15 +7,10 @@ use super::{TransformCache, TransformObject};
 impl TransformObject for Sketch {
     fn transform_with_cache(
         self,
-        transform: &Transform,
-        services: &mut Services,
-        cache: &mut TransformCache,
+        _transform: &Transform,
+        _services: &mut Services,
+        _cache: &mut TransformCache,
     ) -> Self {
-        let faces =
-            self.faces().into_iter().cloned().map(|face| {
-                face.transform_with_cache(transform, services, cache)
-            });
-
-        Self::new(faces)
+        todo!()
```

Comment:
Is this even possible now that sketch is entirely 2d? We can't apply a 3D transform to a 2D sketch?

Reply:
I removed `TransformObject` for `Sketch`, without touching `fj-operations`/`fj` and everything seems to work fine...
Is there anything specific I should check/change?

Reply:
Oh yeah, that's because none of the `Shape::Brep` associated type are actually set to `Sketch`. Since you decided to convert it to 3D using a default plane, no changes will be required.